frontend/src/Components/Login/Login.jsx

```diff
@@ -1,6 +1,117 @@
-import React from "react";
+import React, { useEffect, useRef, useState } from "react";
 import "./Login.css";
+import logoImage from "./assets/logoblackandwhite.svg";
+import AxiosInstance from "../axios/AxiosInstance";
 
-export const Login = () => {
-  return <div>SkyNest</div>;
+const Login = ({ setAccessToken }) => {
+	const [email, setEmail] = useState("");
+	const [password, setPassword] = useState("");
+	const [errorMsg, setErrorMsg] = useState("");
+
+	const emailRef = useRef();
+
+	const getUserToken = async () => {
+		AxiosInstance.post("/login", {
+			email: email,
+			password: password,
+		})
+			.then(({ data }) => {
+				setAccessToken(data.accessToken);
+				localStorage.setItem("accessToken", data.accessToken);
+			})
+			.catch(({ response }) => {
+				if (response.status === 400) {
+					setErrorMsg("Incorrect username or password");
+				} else if (response.status === 0) {
+					setErrorMsg("Server Timeout");
+				}
+				emailRef.current.focus();
+			});
+	};
+
+	useEffect(() => {
+		emailRef.current.focus();
+	}, []);
+
+	useEffect(() => {
+		setErrorMsg("");
+	}, [email, password]);
+
+	const onFormSubmit = (e) => {
+		e.preventDefault();
+		getUserToken();
```

Comment:
Shouldn't you await this function since it is asynchronous? Also, try case when user clicks submit mutliple times what will happen? Maybe it's good to disable submit while waiitng for response from api. Just a thought.

Reply:
Already fiexed the async and await, thanks for the heads up :) 

I know that the user can click multiple times, the goal is to fix that and add a "Loading button as well as disabled inputs" in one of the next tasks.